hdbt.theme

```diff
@@ -1477,6 +1462,76 @@ function hdbt_preprocess_tpr_errand_service(&$variables): void {
   }
 
   $variables['errand_service_additional_info'] = $additional_info;
+
+  $variables['errand_service_channels'] = [];
+
+  // Go through all service channels and give service channel
+  // template required data.
+  foreach ($entity->get('channels')->getValue() as $channel) {
+    $channel = Channel::load($channel['target_id']);
```

Comment:
These changes should be moved here https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/blob/main/src/Entity/ChannelType.php if possible